### PR TITLE
Spanish translation

### PR DESCRIPTION
**Pull request by Daniumy**
Translated a bunch of posts, currently trying to get some more spanish translations since im quite busy with exams but nobody wants to do it"

### CODE REVIEW

File: content/es/blog/2021-11-02-good-whale’s-grant-fund.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+Alex (Kushti) Chepurnoy ha publicado recientemente (https://www.ergoforum.org/t/good-whes-grant-fund/2779) una proposición de un gran inversor de ergo llamado * Buena ballena *.En su deseo de apoyar la investigación y el desarrollo de Ergo, la buena ballena está ofreciendo subvenciones de hasta $ 20,000 SIGUSD (¡cada uno!) este no es un trabajo por contrato, por lo que los desarrolladores que solicitan esta subvención también controlarán los productos creados y cualquier flujo de ingresos resultante.
+
+La ballena buena está interesado principalmente en los siguientes temas:
+
+* Desarrollar un estándar para certificados basados en Nipopow para NFTS
+* Desarrollo de contratos y UI para el Ergo Name System (alternativa para Namecoin, HandShake, Ethereum Name Service)
+* Desarrollar aplicaciones en la parte superior de Ergodex, como DEXYUSD [DEXY USD - STABLECOIN DISEÑO SIMPLEST] (https://www.eroforum.org/t/dexy-usd-simplest-stablecoin-design/1430), Ventas de tokens con garantías de recompra[NUEVO ERGO Smart Dex Pedidos con garantías de recompra |Ergo] (https://ergoplatform.org/en/blog/2020-12-02-new-ergo-smart-dex-orders-with-buyback-guarantes/), etc.
```
Comment (deadit) [inline content/es/blog/2021-11-02-good-whale’s-grant-fund.md:21]: need to merge [NUEVO ERGO Smart Dex Pedidos con garantías de recompra |Ergo] (https://ergoplatform.org/en/blog/2020-12-02-new-ergo-smart-dex-orders-with-buyback-guarantes/)
 to [NUEVO ERGO Smart Dex Pedidos con garantías de recompra |Ergo](https://ergoplatform.org/en/blog/2020-12-02-new-ergo-smart-dex-orders-with-buyback-guarantes/)

File: content/es/blog/2021-11-02-good-whale’s-grant-fund.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+blogPhoto: /img/uploads/good-whale-cover-1-.png
+tags:
+  - Ergo Team
+  - Building Ergo
+---
+<!--StartFragment-->
+
+Ergo cree que las personas deben tomar tanto control del ecosistema posible para apoyar la descentralización y las esperanzas de desarrollar la comunidad criptográfica y el ecosistema a través de los movimientos de base.El ecosistema ERGO está creciendo a un ritmo rápido en términos de desarrollo y base de usuarios.Construimos en colaboración gracias a todos los ergonautas que apoyan y toman el control del ecosistema.
+
+Alex (Kushti) Chepurnoy ha publicado recientemente (https://www.ergoforum.org/t/good-whes-grant-fund/2779) una proposición de un gran inversor de ergo llamado * Buena ballena *.En su deseo de apoyar la investigación y el desarrollo de Ergo, la buena ballena está ofreciendo subvenciones de hasta $ 20,000 SIGUSD (¡cada uno!) este no es un trabajo por contrato, por lo que los desarrolladores que solicitan esta subvención también controlarán los productos creados y cualquier flujo de ingresos resultante.
```
Comment (deadit) [inline content/es/blog/2021-11-02-good-whale’s-grant-fund.md:15]: from "recientemente (https://www.ergoforum.org/t/good-whes-grant-fund/2779)"
to "[recientemente](https://www.ergoforum.org/t/good-whes-grant-fund/2779)"

https://www.markdownguide.org/basic-syntax/#links

**Review by deadit — CHANGES_REQUESTED** [review]

**Comment by deadit** [comment]
Hey, please, check the links in the blog posts. Need to write it like in this example - https://www.markdownguide.org/basic-syntax/#links

File: content/es/blog/2021-11-02-good-whale’s-grant-fund.md (hunk 2 above)
Comment (Daniumy) [reply content/es/blog/2021-11-02-good-whale’s-grant-fund.md:15]: Understood, thanks for teaching me how it s done, will have it fixed asap

**Comment by Daniumy** [comment]
About the last commit: rootZ has already fixed the bit.ly link errors. If you have any questions just let me know through Telegram Dmitry.

**Pull request merged**